sencha/translate.py

```diff
@@ -340,32 +376,36 @@ def get_coding_score_line(self, description, jaccard, n_kmers, special_case, fra
 
     def score_reads_per_file(self, reads):
         """Assign a coding score to each read. Where the magic happens."""
-
-        scoring_lines = []
-
-        with screed.open(reads) as records:
-            for record in tqdm(records):
-                description = record["name"]
-                sequence = record["sequence"]
+        records = []
+        with screed.open(reads) as seqfile:
+            for read in seqfile:
+                description = read.name
                 if self.verbose:
                     logger.info(description)
+                records.append(tuple((description, read.sequence)))
+        n_jobs = self.processes
+        num_records = len(records)
+        if num_records == 1:
+            chunksize = 1
+        else:
+            chunksize = calculate_chunksize(num_records, n_jobs)
 
-                for (
-                    jaccard,
-                    n_kmers,
-                    special_case,
-                    frame,
-                ) in self.maybe_score_single_read(description, sequence):
-                    if self.verbose:
-                        logger.info(
-                            "Jaccard: {}, n_kmers = {} for frame {}".format(
-                                jaccard, n_kmers, frame
-                            )
-                        )
-                    line = self.get_coding_score_line(
-                        description, jaccard, n_kmers, special_case, frame
+        pool = multiprocessing.Pool(processes=n_jobs)
+        logger.info("Pooled %d and chunksize %d mapped", n_jobs, chunksize)
```

Comment:
```suggestion
        logger.info(f"Pooled {n_jobs} and chunksize {chunksize} mapped")
```
just to keep the string formatting consistent

Reply:
thanks, changed